src/components/ContentListTable/ContentListTable.tsx

```diff
@@ -153,15 +161,16 @@ const ContentListTable = () => {
                             name,
                             url,
                             distribution_arch,
-                            distribution_version,
+                            distribution_versions,
                             account_id,
                             org_id
                         }) => (
                             <Tr key={ uuid }>
                                 <Td>{name}</Td>
                                 <Td>{url}</Td>
                                 <Td>{distribution_arch}</Td>
-                                <Td>{distribution_version}</Td>
+                                <Td>{versionDisplay(distribution_versions)}
```

Comment:
if there's a better way to do this than a function, let me know!

Reply:
No, that's totally fine. 
We may want to move the "versionDisplay" method later to a global helper. As we may want to use it in the "Edit" or other views. 
But for now it's okay here.